generators/client/templates/angular/src/main/webapp/app/admin/_admin.module.ts

```diff
@@ -76,7 +76,7 @@ import {
         <%_ if (devDatabaseType !== 'cassandra') { _%>
         AuditsComponent,
         <%_ } _%>
-        <%_ if (!skipUserManagement) { _%>
+        <%_ if (!skipUserManagement && !authenticationType === 'oauth2') { _%>
```

Comment:
wont this be simpler if you just set `skipUserManagement = true` when `authenticationType === 'oauth2'`? you could get rid of a lot of changes in the PR that way

Reply:
Good point. I'll take a look at making this change. Do you know where it might be best to put this logic?

Reply:
I would say on the index.js of server sub gen line 270 below
```
                // user-management will be handled by UAA app
                if (this.applicationType === 'gateway' && this.authenticationType === 'uaa') {
                    this.skipUserManagement = true;
                }
```

Reply:
I went ahead and made this change. However, it took me hours to track down all the other places where I still wanted to do user-related stuff for OAuth and had to add a check for `authenticationType === 'oauth2` back in.